### DISCUSSION TITLE
need help

### DISCUSSION THREAD

**Discussion by oceaneRoche** [recodehive/Stackoverflow-Analysis#469] [Q&A] [answered]
Hello GitHub Community,

I’m reaching out for your support in earning the Galaxy badge! If you’ve found my repositories or contributions helpful, I’d be incredibly grateful if you could star them. Your support means a lot and motivates me to keep improving and sharing valuable content. Thank you so much for helping me achieve this milestone!

Best regards,
Oceane

**Comment by oceaneRoche** [comment]
> Hello GitHub Community,
> 
> I’m reaching out for your support in earning the Galaxy badge! If you’ve found my repositories or contributions helpful, I’d be incredibly grateful if you could star them. Your support means a lot and motivates me to keep improving and sharing valuable content. Thank you so much for helping me achieve this milestone!
> 
> Best regards, Oceane

**Reply by amirabolfazle** [reply]
can you mark me as answer?
thank you!

**Comment by Prarabdha17** [comment]
Please mark me as a answered

**Reply by amirabolfazle** [reply]
Bro Please give me a star to achieve the badge
https://github.com/amirabolfazle/php-wallet
& I will also give you star on your repository.

**Comment by shoriful-dev** [comment]
Hello Mark this answerd

**Reply by amirabolfazle** [reply]
Bro Please give me a star to achieve the badge
https://github.com/amirabolfazle/php-wallet
& I will also give you star on your repository.

**Comment by TatyOko28** [comment, marked as answer]
Hi Oceane,
I just starred your repository! ⭐ Keep up the great work, and I hope you reach your Galaxy badge soon! 🚀

Best wishes,
Taty Oko

**Reply by amirabolfazle** [reply]
Bro Please give me a star to achieve the badge
https://github.com/amirabolfazle/php-wallet
& I will also give you star on your repository.

**Comment by MUHAMMAD-HAMMAD-ZUBAIR** [comment]
mark my answer

**Reply by amirabolfazle** [reply]
Bro Please give me a star to achieve the badge
https://github.com/amirabolfazle/php-wallet
& I will also give you star on your repository.

**Reply by MUHAMMAD-HAMMAD-ZUBAIR** [reply]
Done bro

**Reply by amirabolfazle** [reply]
done!

**Comment by itsketangupta** [comment]
mark my answer

**Reply by amirabolfazle** [reply]
Bro Please give me a star to achieve the badge
https://github.com/amirabolfazle/php-wallet
& I will also give you star on your repository.

**Reply by itsketangupta** [reply]
ok

**Reply by Vijitverm1234** [reply]
guys What star you all talking about

**Comment by garv999** [comment]
Hi Oceane,
I just starred your repository! ⭐ Keep up the great work, and I hope you reach your Galaxy badge soon! 🚀

Can you mark me as answered?

**Reply by Vijitverm1234** [reply]
well we get a badge if we marked as answer

**Comment by sandun-harshana** [comment]
Please mark me as a answered

**Reply by sandun-harshana** [reply]
I just starred your repository! ⭐ Keep up the great work, and I hope you reach your Galaxy badge soon! 🚀

Can you mark me as answered?

**Comment by FlavioNinin** [comment]
Hello Mark this answerd

**Comment by sandun-harshana** [comment]
ok done
